src/frontend/src/hooks/recruitment.hooks.ts

```diff
@@ -0,0 +1,53 @@
+import { useMutation, useQuery, useQueryClient } from 'react-query';
+import { Milestone } from 'shared/src/types/milestone-types';
+import { createMilestone, editMilestone, getAllMilestones } from '../apis/recruitment.api';
+
+export interface CreateMilestonePayload {
```

Comment:
Just for the sake of clarity could you make these `MilestonePayload` and `FaqPayload`